storage/downloading_policy.cpp

```diff
@@ -2,7 +2,21 @@
 
 #include "platform/platform.hpp"
 
-bool StorageDownloadingPolicy::IsDownloadingAllowed() const
+void StorageDownloadingPolicy::EnableCellularDownload(bool value)
+{
+  m_cellularDownloadEnabled = value;
+  m_disableCellularTime = system_clock::now() + hours(1);
```

Comment:
Or, as alternative, use my::Timer from base, it has all what you need to measure duration between two events.